### DISCUSSION TITLE
Calculate direction to align with an astronomical body on the horizon

### DISCUSSION THREAD

**Discussion by d3x0r** [Stellarium/stellarium#3554] [General]
Given a day of year, a latitude and longitude, I'd like to know the direction on the ground to align with a astronomical body with a given declination and right ascension that will appear either on the morning horizon or evening horizon (or both).  I don't really care what time of day the alignment happens, so probably the day of the year doesn't matter.

This is probably something simple I should be able to compute for myself, but I'm drawing blanks.  

![image](https://github.com/Stellarium/stellarium/assets/1909610/1e2cb039-ec3f-47c2-b184-11c73b4c405d)

Something like the above image - at some point on the earth (lat/long), I'd like to know the direction to align to a specific constellation (for example Cetus at +10Dec, 0:30RA ,  or Virgo at -4 Dec, 13 RA  ), I'd like to know the compass direction to align either the red or green line shown from the point.  Basically those stellar points need to be a tangent to the earth's sphere at the point....

**Comment by gzotti** [comment]
You can use the ArchaeoLines plugin to visualize an arbitrary declination line that intersects with the horizon. Or set a marker at your declination and find rise or set times, set this time and read azimuth. Else, outside Stellarium, you need to work out your problem in spherical trigonometry.

**Comment by d3x0r** [comment]
I'm not sure how to set a marker at a specific position - or add a selectable point.
Adding the extra lines from ArchaeoLines gave me a better idea - looks like it ends up being a simple calculation in this case - since the objects are near the equinox, the angles to them on the horizon don't change very much for time of year or position on the earth.

**Reply by gzotti** [reply]
Methinks thou shalt read ye olde User Guide... at least the introduction chapter. With Section 3.6.
